tools/mass_perldoc_md_creation.pl

```diff
@@ -0,0 +1,182 @@
+    'uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm',
+    'uploadNeuroDB/NeuroDB/Notify.pm',
+    'uploadNeuroDB/bin/concat_mri.pl',
+    'uploadNeuroDB/bin/mincpik',
+    'uploadNeuroDB/imaging_upload_file.pl',
+    'uploadNeuroDB/imaging_upload_file_cronjob.pl',
+    'uploadNeuroDB/mass_nii.pl',
+    'uploadNeuroDB/mass_pic.pl',
+    'uploadNeuroDB/minc_deletion.pl',
+    'uploadNeuroDB/minc_insertion.pl',
+    'uploadNeuroDB/register_processed_data.pl',
+    'uploadNeuroDB/tarchiveLoader',
+    'uploadNeuroDB/tarchive_validation.pl'
+);
+
+
+my $profile;
+my $verbose = 0;
+
+my $profile_desc = "name of config file in ../dicom-archive/.loris_mri";
+
+my @opt_table = (
+    [ "-profile", "string",  1, \$profile, $profile_desc ],
+    [ "-verbose", "boolean", 1, \$verbose, "Be verbose." ]
+);
+
+my $Help = <<HELP;
+******************************************************************************
+Creates the perldoc .md files stored in LORIS-MRI/docs/scripts_md/
+******************************************************************************
+
+This script will need to be run once per release to make sure the .md files
+derived from the documentation of the perl scripts and libraries are updated.
+If any new script have been added to a given release, make sure to include it
+in the variable called \@script_list at the beginning of the script.
+
+Documentation: perldoc mass_perldoc_md_creation.pl
+
+HELP
+my $Usage = <<USAGE;
+usage: perl tools/mass_perldoc_md_creation.pl -profile \$profile
+       $0 -help to list options
+USAGE
+&Getopt::Tabular::SetHelp( $Help, $Usage );
+&Getopt::Tabular::GetOptions( \@opt_table, \@ARGV )
+    || exit $NeuroDB::ExitCodes::GETOPT_FAILURE;
+
+
+
+## input error checking
+if ( !$profile ) {
+    print $Help;
+    print STDERR "$Usage\n\tERROR: missing -profile argument\n\n";
+    exit $NeuroDB::ExitCodes::PROFILE_FAILURE;
+}
+{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
+if ( !@Settings::db ) {
+    print STDERR "\n\tERROR: You don't have a \@db setting in the file "
+        . "$ENV{LORIS_CONFIG}/.loris_mri/$profile \n\n";
+    exit $NeuroDB::ExitCodes::DB_SETTINGS_FAILURE;
+}
+
+
+
+## database connection
+my $dbh = &NeuroDB::DBI::connect_to_db(@Settings::db);
+print "\n==> Successfully connected to database \n" if $verbose;
+
+
+
+## get data directory from the Config table
+my $loris_mri_path = NeuroDB::DBI::getConfigSetting(\$dbh, 'MRICodePath');
+$loris_mri_path    =~ s/\/$//;
+
+
+
+## add the LORIS-MRI directory path to the list of scripts
+@script_list = map { $loris_mri_path . '/' . $_ } @script_list;
+
+
+
+## create the list of md file based on @script_list and change extension to .md
+my $md_path  = $loris_mri_path . '/docs/scripts_md/';
+my @suffixes = (".pl", ".pm");
+my @md_list  = map { $md_path . basename($_, @suffixes) . ".md"} @script_list;
+
+
+
+## loop through script array and create the .md files using pod2markdown
+my $git_add = "git add";
```

Comment:
A note for me:
it is fair to assume that `git` is installed since this is how the `readme` tells projects to install Loris-MRI,
so I think we r all good here